code/modules/mob/living/living.dm

```diff
@@ -873,7 +873,7 @@ Thanks.
 	else if(istype(src.loc, /obj/item/delivery/large)) //Syndie item
 		var/obj/item/delivery/large/package = src.loc
 		to_chat(L, "<span class='warning'>You attempt to unwrap yourself, this package is tight and will take some time.</span>")
-		if(do_after(src, src, 100))
```

Comment:
shouldn't it be `do_after(L, src, 1200)`? Also use `2 MINUTES` instead

Reply:
``src`` is the same as ``L`` here anyway, right? You mean ``do_after(L, package, 2 MINUTES)``?

Reply:
yes that's what I meant, I didn't bother to scroll up